Open_Weather-in-hook-and-redux/src/components/common/Info/Info.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react';
+import { getTime } from '../../utils/getTime';
+import { getDate } from '../../utils/getDate';
+import { Weather } from '../../../types/Weather';
+import css from './info.module.css';
+
+
+
+
+export const Info: FC<{weather: Weather }> = (weather) => {
```

Comment:
любой компонент принимает пропсы, погода будет лежать в props.weather
```suggestion
export const Info: FC<{weather: Weather }> = (props) => {
```